src/hotspot/share/gc/shenandoah/shenandoahHeap.cpp

```diff
@@ -72,6 +73,8 @@
 #include "gc/shenandoah/mode/shenandoahIUMode.hpp"
 #include "gc/shenandoah/mode/shenandoahPassiveMode.hpp"
 #include "gc/shenandoah/mode/shenandoahSATBMode.hpp"
+#include "gc/shenandoah/shenandoahScanRemembered.inline.hpp"
```

Comment:
I noticed now, that you also include shenandoahScanRemembered.hpp above. You only need to include the .inline.hpp, and bonus points for keeping it in alphabethical order... ;-)

Reply:
Thanks.  This fix will be reflected in my next push.